2255.count-prefixes-of-a-given-string.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/count-prefixes-of-a-given-string/
+ */
+
+/**
+ * @param {string[]} words
+ * @param {string} s
+ * @return {number}
+ */
+var countPrefixes = function(words, s) {
+    
+    const changeToArr = s.toString("");
```

Comment:
```suggestion
```
This line seems unnecessary. 
You can use "s" as it is instead of "changeToArr" in the lines below.